test/seeds/index.ts

```diff
@@ -39,6 +41,16 @@ export const seed = async () => {
     },
   ])
 
+  await db.transaction().insert([
+    {
+      id: seededTransactionId,
+      token_type: 'DEMAND',
```

Comment:
you can import the `TokenType` enum and use that instead of writing out the string, same with `state`